headers/augment_headers.py

```diff
@@ -96,6 +96,7 @@ def format_file(self, filename: str):
 class HeaderAugmenter:
     DOCSTRING_PREAMBLE_LINE = "// THIS DOCSTRING WAS GENERATED AUTOMATICALLY\n"
     CPP_COMMENT_RE = re.compile(r"\s*//")
+    DEPRECATED_MACRO_GUARD_RE = re.compile(r"#define HEADERS_[A-Z0-9_]+_H_?")
```

Comment:
Let's call this `INCLUDE_GUARD_RE`. It's not matching against the DEPRECATED macro, it's matching against the normal include guard.